### ISSUE TITLE
Live redirect issue after gitpull

### ISSUE THREAD

**Issue by milehighkaraoke** [WWBN/AVideo#7769]
![Screenshot_20230308_231345_Chrome](https://user-images.githubusercontent.com/70327833/223936569-14c864f2-923c-49e1-b143-f14b3683ebde.jpg)

**Comment by milehighkaraoke** [comment]
![Screenshot_20230308_233858_Chrome](https://user-images.githubusercontent.com/70327833/223941034-8166a0a1-874b-4151-a5c7-db6f6130f3bd.jpg)

**Comment by milehighkaraoke** [comment]
[08-Mar-2023 23:37:13 America/Dawson_Creek] AVideoLog::SECURITY: NGINX ON Publish denied  {"error":true,"liveTransmitionHistory_id":0} SCRIPT_NAME: /plugin/Live/on_publish.php

**Comment by milehighkaraoke** [comment]
Also lots of photos errors. Here are the first few. The repeat
![Screenshot_20230308_234501_Chrome](https://user-images.githubusercontent.com/70327833/223942340-375de527-1689-40bd-ab87-d5867e1b770d.jpg)
![Screenshot_20230308_234515_Chrome](https://user-images.githubusercontent.com/70327833/223942345-a1700073-d153-4e2f-9ce1-221279892b91.jpg)

**Comment by milehighkaraoke** [comment]
![Screenshot_20230308_234840_Chrome](https://user-images.githubusercontent.com/70327833/223942913-3a3e32ed-68bb-4781-8c2f-8244c2bf5ed2.jpg)

**Comment by DanielnetoDotCom** [comment]
The redirect issue has nothing to do with AVideo code, maybe something you changed in your server or DNS.

The record and the warnings seem to be missing permission issues. By default, that is the command that fixes it:

```

chown -R www-data:www-data /var/www/html

chmod -R 777 /var/www/html/AVideo/videos

chmod -R 777 /var/www/tmp

```

The `/var/www/tmp` directory is for the recording, and this path is set in the `/usr/local/nginx/conf/nginx.conf`.

Note that you may need to change the paths to match your installation directory.

**Comment by milehighkaraoke** [comment]
I didn't change anything. All I did was a gitpull.

**Comment by milehighkaraoke** [comment]
Redirect works for saved videos

**Comment by milehighkaraoke** [comment]
@DanielnetoDotCom 
Did the new update change the configuration files of nginx and remove permissions? I do absolutely nothing to this box but the git pulls. I have also noticed that my site shows all txt for a split second before the site appears.

**Comment by DanielnetoDotCom** [comment]
> @DanielnetoDotCom
> Did the new update change the configuration files of nginx and remove permissions? I do absolutely nothing to this box but the git pulls. I have also noticed that my site shows all txt for a split second before the site appears.


I apologize, but I'm not sure what might be causing the issue you're experiencing. Additionally, I'm not familiar with the redirect behavior you mentioned. If I'm unable to replicate the error, I may not be able to provide assistance.

If you'd like more direct assistance, you can consider hiring me through the [YouPHPTube marketplace](https://youphp.tube/marketplace/). However, please understand that while I will do my best to help, I cannot guarantee a solution.

**Comment by milehighkaraoke** [comment]
Any idea @DanielnetoDotCom 
![Uploading Screenshot_20230309_103507_Chrome.jpg…]()

**Comment by milehighkaraoke** [comment]
@DanielnetoDotCom 
Did port's and or path change on the new git pull?
It's like something got reset after the git pull
![Screenshot_20230309_103507_Chrome](https://user-images.githubusercontent.com/70327833/224110144-3013637a-5976-4080-9bee-eb57307273c4.jpg)

**Comment by milehighkaraoke** [comment]
Still getting 1000s of php errors
![Uploading Screenshot_20230309_104021_Chrome.jpg…]()

**Comment by milehighkaraoke** [comment]
Still getting 1000s of php errors
![Screenshot_20230309_104021_Chrome](https://user-images.githubusercontent.com/70327833/224110883-fb74c6ae-204d-4265-a662-12443d772d11.jpg)

**Comment by GreenDragonTavern** [comment]
@milehighkaraoke - on your web browser(s) try clearing cookies and cache. Firefox was giving me major grief too after git pull. Both on mobile and laptop.

**Comment by milehighkaraoke** [comment]
@GreenDragonTavern it's also effecting my live embedded streams so it's not a browser problem  but I did try cache with no resolve

**Comment by GreenDragonTavern** [comment]
> ...I have also noticed that my site shows all txt for a split second before the site appears.

I should have been more specific, it was regarding the display of all text on browser(s).

**Comment by DanielnetoDotCom** [comment]
Can you reproduce it with the demo site?

**Comment by milehighkaraoke** [comment]
@DanielnetoDotCom
OK I found the redirect issue.  Seems that after git pull it added www in front of mhktube in all my strings in the live plugin. I removed the www. And now redirect is fixed. What can I do to assure this does not happen again when I do a gitpull?
![Screenshot_20230309_214642_Firefox](https://user-images.githubusercontent.com/70327833/224226405-29e43f39-9196-437c-a330-54fc8e2e252b.jpg)
